MapboxNavigation/FeedbackViewController.swift

```diff
@@ -226,30 +224,49 @@ public class FeedbackViewController: UIViewController, DismissDraggable, UIGestu
         
         NSLayoutConstraint.activate(constraints)
     }
-    
+
+    /**
+     Sends the current feedback event for the specific feedback item.
+
+     - Parameter item: a feedback item for which feedback will be sent.
+     */
     func send(_ item: FeedbackItem) {
-        if let uuid = self.uuid {
+        if let uuid = self.currentFeedbackEventID() {
             delegate?.feedbackViewController(self, didSend: item, uuid: uuid)
             eventsManager?.updateFeedback(uuid: uuid, type: item.feedbackType, source: .user, description: nil)
         }
-        
+
         guard let parent = presentingViewController else {
             dismiss(animated: true)
             return
         }
-        
+
         dismiss(animated: true) {
             DialogViewController().present(on: parent)
         }
     }
-    
+
+    /**
+     Cancels sending the current feedback event.
+     */
     func dismissFeedbackItem() {
         delegate?.feedbackViewControllerDidCancel(self)
-        if let uuid = self.uuid {
+        if let uuid = self.currentFeedbackEventID() {
             eventsManager?.cancelFeedback(uuid: uuid)
         }
         dismiss(animated: true, completion: nil)
     }
+
+    /**
+     Gets an unique identifier used to identify the current feedback event.
+
+     The UUID that method returns is the result of calling `NavigationEventsManager.recordFeedback()`.
+
+     - Returns: UUID used to identify the current feedback event.
+     */
+    @discardableResult private func currentFeedbackEventID() -> UUID? {
```

Comment:
This method’s name should indicate that it has a side effect, not just that it returns a UUID. `recordFeedback()` would be appropriate here.